utils/docker/Dockerfile-rpmbuild.sles.12.3

```diff
@@ -0,0 +1,116 @@
+#
+# Copyright 2019, Intel Corporation
+#
+# 'recipe' for Docker to build an RPM
+#
+
+# Pull base image
+FROM suse/sles:12.3
+MAINTAINER Brian J. Murrell <brian.murrell@intel.com>
+
+# use same UID as host and default value of 1000 if not specified
+ARG UID=1000
+
+# Add build user (to keep rpmbuild happy)
+ENV USER build
+ENV PASSWD build
+RUN useradd -u $UID -ms /bin/bash $USER
+RUN groupadd -g $UID $USER
+RUN echo "$USER:$PASSWD" | chpasswd
+
+# Install basic tools
+RUN zypper --non-interactive update
+# basic building
+RUN zypper --non-interactive install make rpm-build curl createrepo git    \
+                                     lsb-release autoconf automake libtool \
+                                     ca-certificates-mozilla
+# libfabric
+RUN zypper --non-interactive install rdma-core-devel libnl3-devel        \
+                                     infinipath-psm-devel valgrind-devel
+# mercury
+RUN zypper --non-interactive ar --gpgcheck-allow-unsigned -f https://build.hpdd.intel.com/job/daos-stack/job/openpa/job/master/lastSuccessfulBuild/artifact/artifacts/sles12.3/ openpa;       \
+    zypper --non-interactive ar --gpgcheck-allow-unsigned -f https://build.hpdd.intel.com/job/daos-stack/job/libfabric/job/master/lastSuccessfulBuild/artifact/artifacts/sles12.3/ libfabric; \
+    zypper --non-interactive ref openpa libfabric
+# our libfabric conflicts with libfabric1
+# TODO: consider if we should rename ours or replace libfabric1, etc.
+RUN if rpm -q libfabric1; then zypper --non-interactive remove libfabric1; fi
+RUN zypper --non-interactive install gcc-c++
+RUN zypper --non-interactive --no-gpg-check install openpa-devel      \
+                                                    libfabric-devel   \
+                                                    cmake boost-devel
+# pmix
+RUN zypper --non-interactive install libevent-devel
+
+# ompi
+RUN zypper --non-interactive ar -f https://download.opensuse.org/repositories/science:/HPC:/SLE12SP3_Missing/SLE_12_SP3/ hwloc;                                                     \
+    zypper --non-interactive --gpg-auto-import-keys ref hwloc;                                                                                                                      \
+    zypper --non-interactive ar --gpgcheck-allow-unsigned -f https://build.hpdd.intel.com/job/daos-stack/job/pmix/job/master/lastSuccessfulBuild/artifact/artifacts/sles12.3/ pmix; \
```

Comment:
Need to have the URL passed for the current Jenkins.

Reply:
Fixed.